js/main.js

```diff
@@ -7,23 +7,37 @@ var entries = document.querySelector('ul');
 var entriesAnchor = document.querySelector('#entries-anchor');
 var newButton = document.querySelector('.new-button');
 var views = document.querySelectorAll('.view');
+var newEditEntry = document.querySelector('.new-edit-entry');
+var edit = false;
 
 switchViews(data.view);
 
 function saveEntry(event) {
   event.preventDefault();
-  var entry = {};
-  entry.title = form.elements.title.value;
-  entry.photoUrl = form.elements['photo-url'].value;
-  entry.notes = form.elements.notes.value;
-  entry.id = data.nextEntryId;
-  data.nextEntryId++;
-  data.entries.unshift(entry);
-  img.setAttribute('src', 'images/placeholder-image-square.jpg');
-  entries.prepend(loadEntry(entry));
+  if (edit === false) {
```

Comment:
Instead of making a new variable for this, you should just check if there is a truthy value in `data.editing`. If you reset `data.editing` back to `null` after every time the form is submitted, you can always count on only needing to do editing work if there is a value in `data.editing`.